<launch>

  <param name="Tuning" value="True" type="bool" />
  <param name="lowerH" value="0" type="int" />
  <param name="lowerS" value="0" type="int" />
  <param name="lowerV" value="120" type="int" />
  <param name="upperH" value="195" type="int" />
  <param name="upperS" value="255" type="int" />
  <param name="upperV" value="255" type="int" />

  <node name="nao_tomato_tracker_node" pkg="nao_tomato_tracker" type="nao_tomato_tracker_node" output="screen">
    <remap from="/image_raw" to="/nao_camera/image_raw"/>
  </node>

</launch>
